Automattic/jetpack
# SSO: Add sso disable modal

Fixes https://github.com/Automattic/wp-calypso/issues/88410

This PR enables us to gather more data on the reasons why users are disabling the SSO module.
When a user disables the SSO module, we show a modal that leads to a crowsignal survey.

Design: eqlepL7MHifK8Bgr1OWqLr-fi-15013_60324

<img width="741" alt="CleanShot 2024-03-19 at 14 26 43@2x" src="https://github.com/Automattic/jetpack/assets/10482592/4be23edf-97f1-464b-94cf-d9291b448326">

<img width="739" alt="CleanShot 2024-03-19 at 14 27 22@2x" src="https://github.com/Automattic/jetpack/assets/10482592/7b1d0ab9-b7b8-4656-8bab-d1907983d20b">



## Proposed changes:

* Show survey modal when disabling SSO

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync this PR to your site
* Go to /wp-admin/admin.php?page=jetpack#/settings
* A modal should show when you disable the SSO module
* Take the survey
* Verify the correct userId is added to the survey URL
* Verify the survey modal does not display multiple times if you disable/enable the SSO module
